.github/workflows/docker.yaml

```diff
@@ -0,0 +1,355 @@
+# - Performs a virus scan.
+# - Pushes the container to a registry.
+# - If triggered by a prerelease, this workflow publishes the release when it's done. (This triggers the deploy workflow.)
+# - If triggered by workflow_dispatch:
+#   - Takes an optional git ref for sources to build from.
+#   - Takes an optional image tag name.
+#   - Optionally triggers deploy when done.
+
+name: Docker
+
+on:
+  workflow_call:
+    inputs:
+      image_name:
+        description: Space-separated list of image base names
+        type: string
+        required: true
+      build_command:
+        description: Command to run to build docker image. `${DOCKER_FULL_NAME}` should be used as the tag name.
+        type: string
+        required: false
+        default: ""
+      buildlog_repo:
+        type: string
+        required: false
+        default: ""
+      buildlog_filename:
+        type: string
+        required: false
+        default: "${{ github.repository }}.json"
+      additional_decryption_files:
+        description: Space-separated list of additional `.iron` files to decrypt. Gcloud credentials will already be decrypted.
+        type: string
+        required: false
+        default: ""
+      archs:
+        description: Space-separated list of architectures to build.
+        type: string
+        required: false
+        default: amd64
+      docker_runs_on:
+        description: JSON list of runners to use for `docker` job within the workflow.
+        type: string
+        required: false
+        default: '["buildjet-2vcpu-ubuntu-2204"]'
+      other_jobs_runs_on:
+        description: Runner to use for `image` and `publish` jobs within the workflow.
+        type: string
+        required: false
+        default: "ubuntu-22.04"
+      configure_git_ssh:
+        description: Whether to configure git ssh with LEEROY_PRIVATE_SSH_KEY repository secret.
+        type: boolean
+        required: false
+        default: false
+      setup_scala_java_version:
+        description: If set, calls setup-scala with the provided java version number before building.
+        type: string
+        required: false
+        default: ""
+      extra_commands:
+        description: Extra commands to run before building the Docker image
+        type: string
+        required: false
+        default: ""
+      skip_update_release:
+        description: "`true` to skip marking the triggering release as `prerelease: false`."
+        type: boolean
+        required: false
+        default: false
+      docker_build_target:
+        description: "Extra target flag to pass to the `docker build` command. Note that this should include the `--target` as well."
+        type: string
+        required: false
+        default: ""
+
+env:
+  DOCKER_BUILDKIT: 1
+
+jobs:
+  image:
+    if: github.actor != 'dependabot[bot]'
+    runs-on: ${{ inputs.other_jobs_runs_on }}
+    env:
+      DOCKER_IMAGE: ${{ inputs.image_name }}
```

Comment:
```suggestion
```

I think we should delete this because it's only referenced in one place below, and its needless indirection makes things slightly harder to follow.